toolchain/binutils: remove old versions

zpc0 opened the pull request:
Reduces supported versions to 3.

hauke commented on toolchain/binutils/Config.in:
```diff
@@ -6,22 +6,6 @@ choice
-		bool "Binutils 2.39"
-		select BINUTILS_VERSION_2_39
-
-	config BINUTILS_USE_VERSION_2_40
```
Please drop all older versions than the one currently used by default except the version which was used in the previous release.
Please keep version 2.40, 2.42 and 2.43 and remove version 2.41 instead.

zpc0 replied:
I updated the commit.
Thank you for review!

zpc0 committed:
toolchain/binutils: remove old versions

Reduce supported versions to 3.
Since 2.40 is supported by previous release, this commit
keeps it.

Signed-off-by: Yanase Yuki <dev@zpc.st>
Link: https://github.com/openwrt/openwrt/pull/16597
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
